### PR TITLE
[Addition] Implemented Getter, fixed router names and virtual scroller

### LINKED ISSUE #56: Verify & Implement Getter: items/getItemsByCategory
#### Function to implement:

Getter: items/getItemsByCategory

Things to note:

-   [ ] Ensure different branch for each issue.
-   [ ] Rebase frequently to avoid conflicts later on
-   [ ] Try to ensure only a couple of commits
-   [ ] Test your code thoroughly before pushing
-   [ ] Add to description, "Fixes #issue-no, eg: Fixes #x (where #x is the issue id)
-   [ ] Work to be done on template branch. i.e. you need to checkout the template branch on your local computer and PR should be to the template branch

### PR DESCRIPTION
**Pull request by Megha-Bose**
Fixes #56

### CODE REVIEW

**Comment by Pk13055** [comment]
Have you verified items are being shown from the backend?

**Comment by Megha-Bose** [comment]
Yes.

File: frontend/src/views/CategoryPage.vue [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <template>
-  <v-content>
-    <ItemsDisplay />
-  </v-content>
+  <ItemsDisplay />
```
Comment (Pk13055) [inline frontend/src/views/CategoryPage.vue:2]: As far as I know, for proper rendering, you have to wrap the component in a `v-content` as per vuetify guidelines. Any reason as to why you have changed this?

Comment (Megha-Bose) [reply frontend/src/views/CategoryPage.vue:2]: I did that to remove that extra white space before contents are displayed.

Comment (Pk13055) [reply frontend/src/views/CategoryPage.vue:2]: Okay I see

**Pull request merged**